Resources/Prototypes/Entities/Objects/Misc/candy_bowl.yml

```diff
@@ -0,0 +1,68 @@
+- type: entity
+  parent: BaseStructureDynamic
+  id: CandyBowl
+  name: "candy bowl"
+  description: Grab as much as you can fit in your pockets!
```

Comment:
Why not BaseItem?

Reply:
This was copy pasted from the paper bin, since that's the behaviour we wanted to emulate, but for candy.
I don't know why not BaseItem, but I suppose now I'll look into it.